reference-manual/src/main/asciidoc/pipeline-api/pipeline-api.adoc

```diff
@@ -196,7 +196,7 @@ enrichment_.
 
 Jet offers two basic techniques to enrich your data stream:
 
-. <<hash-join, Hash-join>> the stream with one or more datasets that you
+. <<hash join, Hash-join>> the stream with one or more datasets that you
```

Comment:
This should be the other way: 
```
<<hash-join, Hash join>>
```

Reply:
Yes, I would broke the link... but i have left `Hash-join` as a verb.